### ISSUE TITLE
Weird issue resolving dep in AS after using KI

### ISSUE THREAD

**Issue by fluxxion82** [Kotlin/kotlin-interactive-shell#124]
Ok, so I'm this weird issue and not entirely sure what's going on, and hoping here can give me some insights.

I was doing some work in Android Studio, and wanted to test something out related to kotlinx datetime quickly, so I go to the command line and open KI. I try to add the repository, dependsOn, and import classes but couldn't get things to work right. This was the first time I had tried adding a maven repo and trying to access a library with KI. I never got it to work so I went to Intellij and did my test. 

I go back to AS and it now can't resolve the kotlinx datetime library anymore (specifically for the version I was trying to use in KI). I cleared my gradle caches, uninstalled KI, reboot my computer but nothing has worked. 

I added a `resolutionStrategy` in my build file to force the project to use an earlier version of kotlinx datetime and that gets the project working again, but I can't use the latest version, or at least the version i was using previously.  My intellij project seems to be working ok with the latest datetime library.

The error in AS that I get is `Unresolved reference` errors. I'm using AS Flamingo Beta 4 (also have issues with Giraffe). On a mac m1.

I'm at a loss of what the issue could be or what else I can do. I'm wondering if there's anyway that KI could have affected what I'm seeing with AS, and if anyone has any ideas what could be causing the issue.

**Comment by fluxxion82** [comment]
I finally figured out the correct repository url/dependsOn value to use (-jvm), but it hasn't helped AS from resolving the classes.

**Comment by davidedelerma** [comment]
Having the same issue. Any luck?

**Comment by fluxxion82** [comment]
> Having the same issue. Any luck?

No, it still happens if I use the particular version of the library. I don't hit the issue too often since there are newer versions of the library but it's definitely still a problem if I use code referencing the old lib version.
